shipgen/MuDISGenerator.h

```diff
@@ -42,12 +42,17 @@ class MuDISGenerator : public FairGenerator
   Double_t startZ,endZ;
   TClonesArray* iMuon ;
   TClonesArray* dPart ;
+  TClonesArray* dPartSoft ;
   FairLogger*  fLogger; //!   don't make it persistent, magic ROOT command
   TFile* fInputFile;
   TTree* fTree;
   int fNevents;
   int fn;
   bool fFirst;
+
+  const Double_t c_light = 29.9792458;//speed of light in cm/ns
+  const Double_t muon_mass = 0.10565999895334244;  //muon mass in GeV
```

Comment:
Well, I'm not certain if these are available as constants elsewhere, I see that for GenieGenerator, c_light is used without definition for calculations. This could cause lack of clarity (timing bug in neuDIS), so I had them in the header file, instead of the implementation file.

Reply:
It's fine to have them in this header, the question was rather why these are member variables/attributes rather than normal variables.